### PR TITLE
Update README with "how to contact us" guide

### PR DESCRIPTION
**Pull request by AlenaSviridenko**
# Description
Updated README with information about usage of discussions

image

## Check list
- [n\a] Related issue / work item is attached
- [n\a] Tests are written (if applicable)
- [n\a] Documentation is updated (if applicable)
- [n\a] Changes are tested and related VM images are successfully generated

### CODE REVIEW

**Review by al-cheb — APPROVED** [review]

**Review by maxim-lobanov — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
...
+This repository contains the source used to create the [virtual environments](https://help.github.com/en/actions/reference/virtual-environments-for-github-hosted-runners) for GitHub Actions hosted runners, as well as the VM images of [Microsoft-hosted agents](https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/hosted?view=azure-devops#use-a-microsoft-hosted-agent) used for Azure Pipelines. To build a VM machine from this repo's source, see the [instructions](./help/CreateImageAndAzureResources.md).
+
+How to get in touch with us:
+- To file bug report, or request tools to be added/updated, please [open an issue using the appropriate template](https://github.com/actions/virtual-environments/issues/new/choose)
```
Comment (miketimofeev) [inline README.md:5]: To file **a** bug report or request tools

File: README.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
...
+This repository contains the source used to create the [virtual environments](https://help.github.com/en/actions/reference/virtual-environments-for-github-hosted-runners) for GitHub Actions hosted runners, as well as the VM images of [Microsoft-hosted agents](https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/hosted?view=azure-devops#use-a-microsoft-hosted-agent) used for Azure Pipelines. To build a VM machine from this repo's source, see the [instructions](./help/CreateImageAndAzureResources.md).
+
+How to get in touch with us:
+- To file bug report, or request tools to be added/updated, please [open an issue using the appropriate template](https://github.com/actions/virtual-environments/issues/new/choose)
+- If you want to share your thoughts about image configuration, installed software or bring some idea, please, create a new topic in a [discussions section](https://github.com/actions/virtual-environments/discussions) for a corresponding category. Prior to making a new discussion please make sure no similar topics were created earlier.
```
Comment (miketimofeev) [inline README.md:6]: about image configuration, installed software, or bring some idea

Comment (miketimofeev) [inline README.md]: Prior to -> Before? To sound more friendly 😃

File: README.md (hunk 1 above)
Comment (AlenaSviridenko) [reply README.md]: done, thanks!

File: README.md (hunk 2 above)
Comment (AlenaSviridenko) [reply README.md]: I used NPM discussions text as an example, but ok, I agree

**Pull request merged**